css/styles.css

```
@@ -0,0 +1,214 @@
+:root {
+    --pale-blue: hsl(243, 100%, 80%);
+    --grayish-blue: hsl(229, 7%, 55%);
+    --dark-blue: hsl(228, 56%, 26%);
+    --very-dark-blue: hsl(229, 57%, 11%);
+}
+
+body {
+    font-family: 'Raleway', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    color: var(--pale-blue);
+
+    background-color: var(--very-dark-blue);
+    background-repeat: no-repeat;
+
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+/*
+    Utilities
+*/
+.has-darkblue-bg {
+    background-color: var(--dark-blue);
+}
+
+.bold {
+    font-weight: 700;
+}
+
+.big {
+    font-size: 3em;
+}
+
+/*
+    First Box
+*/
+.action {
+    max-width: 10em;
+    border-radius: 10px 100px 10px 10px;
+    padding: 2.8em 9em 2.8em 2.8em;
+    margin: 1.1em;
+}
+
+.brand_logo {
+    margin-bottom: 2em;
+}
+
+.action_control {
+    display: flex;
+    justify-content: space-between;
+}
+
+.action_control_button {
+    background-color: var(--very-dark-blue);
+    padding: 1em;
+    border-radius: 10px;
+    display: flex;
+    align-items: end;
+}
+.action_control_button:nth-of-type(2n) {
+    margin: 0 1.5em;
+}
+
+/*
+    Second Box
+*/
+.usage_container {
+    margin: 0;
+    padding: 0;
+}
+
+.usage {
+    border-radius: 10px;
+    padding: 3em;
+    margin: 1.1em;
+    position: relative;
+}
+
+.usage_scale {
+    display: flex;
+    justify-content: space-between;
+}
+
+.usage_pop {
+    background-color: #FFF;
+    color: var(--very-dark-blue);
+    border-radius: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 180px;
+    max-width: 180px;
+    min-height: 72px;
+    max-height: 72px;
+}
+
+.usage_pop_text {
+    color: var(--grayish-blue);
+    margin-left: 0.5em;
+    text-transform: capitalize;
+}
```

```suggestion
word-spacing: 0.2em;
}
```
Je pense que le texte est plus aéré dans le design donné